trollflow2/plugins/__init__.py

```diff
@@ -128,32 +128,46 @@ def resample(job):
                                      conf)
         LOG.debug('Resampling to %s', str(area))
         if area == 'None':
-            minarea = get_config_value(product_list,
-                                       '/product_list/areas/' + str(area),
-                                       'use_min_area')
-            maxarea = get_config_value(product_list,
-                                       '/product_list/areas/' + str(area),
-                                       'use_max_area')
-            native = conf.get('resampler') == 'native'
-            if minarea is True:
-                job['resampled_scenes'][area] = scn.resample(scn.min_area(),
-                                                             **area_conf)
-            elif maxarea is True:
-                job['resampled_scenes'][area] = scn.resample(scn.max_area(),
-                                                             **area_conf)
-            elif native:
-                job['resampled_scenes'][area] = scn.resample(resampler='native')
-            else:
-                # The composites need to be created for the saving to work
-                if not set(scn.keys()).issuperset(scn.wishlist):
-                    LOG.debug("Generating composites for 'null' area (satellite projection).")
-                    scn.load(scn.wishlist, generate=True)
-                job['resampled_scenes'][area] = scn
+            job['resampled_scenes'][area] = _resample_null_area(product_list, scn, conf, area_conf)
```

Comment:
Is there any documentation for the `area == 'None'` behaviour?

Reply:
The only place I could find is in https://github.com/pytroll/trollflow2/blob/main/examples/pl.yaml#L114

We'd really need to add some proper documentation and not hide things only in (commented-out in this case) example configuration files.

Reply:
In the example it says `null` though, not `None`?

Reply:
In yaml `null` is used to denote `None` of Python. I don't remember why in the code we change `None` to `'None'` though...

Reply:
do we have a test for that?